MultiPage-Clone/css/project.css

```diff
@@ -326,6 +334,139 @@ ul {
    }
 }
 
+@media screen and (min-width: 900px) {
```

Comment:
Will complicate a bit the task for you)
Do all the properties below change comparing to other screen sizes?
If not it's better to have them outside of the media query block.
Less code to maintain and to load to browser.

Reply:
I saw you deleted some pieces of media queries. Well done. Keep going